recipes/capstone/all/conanfile.py

```diff
@@ -0,0 +1,83 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class CapstoneConan(ConanFile):
+    name = "capstone"
+    license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://www.capstone-engine.org"
+    description = "Capstone disassembly/disassembler framework: Core (Arm, Arm64, BPF, EVM, M68K, M680X, MOS65xx, Mips, PPC, RISCV, Sparc, SystemZ, TMS320C64x, Web Assembly, X86, X86_64, XCore) + bindings."
+    topics = ("conan", 'reverse-engineering', 'disassembler', 'security', 'framework', 'arm', 'arm64', 'x86', 'sparc', 'powerpc', 'mips', 'x86-64', 'ethereum', 'systemz', 'webassembly', 'm68k', 'm0s65xx', 'm680x', 'tms320c64x', 'bpf', 'riscv')
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False],
+               "fPIC": [True, False],
+               "use_default_alloc": [True, False]}
```

Comment:
This option is not used 🤔

Reply:
yes, added option, but forget about CMake variable :( now added